tapgarden/seedling.go

```diff
@@ -181,6 +181,14 @@ func (c Seedling) validateGroupKey(group asset.AssetGroup,
 			"group asset type %v", group.Genesis.Type)
 	}
 
+	return validateAnchorMeta(c.Meta, anchorMeta)
+}
+
+// validateAnchorMeta checks that the metadata of the seedling matches that of
+// the group anchor, if there is a group anchor.
+func validateAnchorMeta(seedlingMeta *proof.MetaReveal,
```

Comment:
We're not restricting seedlings to genesis canonical URLs here anymore, which I think is correct and what we need.